tests/integration/targets/homebrew_services/tasks/main.yml

```diff
@@ -0,0 +1,94 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- when: ansible_facts.distribution == 'MacOSX'
+  block:
```

Comment:
a good way not to have to add all your code inside this block, is to, instead, begin this file with:
```
- meta: end_play
  when: ansible_facts.distribution != 'MacOSX'
```

Reply:
Lovely!